akvo/templates/myrsr/project_editor.html

```diff
@@ -48,7 +48,8 @@
             </div>
         </div>
         {% trans 'RSR editors can edit the project information, and RSR administrators can edit and publish the project once all mandatory fields are filled out and the progress bar on top of the page is green. Changes that are made to a published project, will be visible online immediately after saving.' %}<br/><br/>
-        {% trans 'If you have questions, please get in touch with us via' %} <a href="mailto:support@akvo.org">support@akvo.org</a>.
+        {% trans 'If you have questions, please get in touch with us via' %}
+        <script type="text/javascript">document.write(String.fromCharCode(60,97,32,104,114,101,102,61,34,109,97,105,108,116,111,58,114,115,114,64,97,107,118,111,46,111,114,103,63,115,117,98,106,101,99,116,61,82,83,82,32,102,101,101,100,98,97,99,107,34,62,114,115,114,64,97,107,118,111,46,111,114,103,60,47,97,62));</script>.
```

Comment:
I'm a bit hesitant to have the support mailto as a document.write(). What if the poor user has some problem with JS and wants to report? :stuck_out_tongue: 
Also I think there should be a django template comment documenting what we're actually outputting with the document.writes